sources/PseudoTerminal.m

```diff
@@ -6124,7 +6144,7 @@ - (void)safelySetSessionSize:(PTYSession*)aSession rows:(int)rows columns:(int)c
     }
     PtyLog(@"safelySetSessionSize");
     BOOL hasScrollbar = [self scrollbarShouldBeVisible];
-    if (windowType_ == WINDOW_TYPE_NORMAL || windowType_ == WINDOW_TYPE_NO_TITLE_BAR) {
```

Comment:
Did you mean to remove this? It would prevent term-initiated resizes from working for no-titlebar windows.